katello/katello.spec

```diff
@@ -85,6 +89,13 @@ install -m 755 %{SOURCE3} %{buildroot}%{_sysconfdir}/cron.weekly/katello-remove-
 install -m 755 %{SOURCE10} %{buildroot}%{_sysconfdir}/cron.weekly/katello-clean-empty-puppet-environments
 install -m 755 %{SOURCE12} %{buildroot}%{_sysconfdir}/cron.daily/katello-repository-publish-check
 
+# symlink script libraries
+mkdir -p %{_datarootdir}/katello
```

Comment:
It seems you need to prefix these with `%{buildroot}` see the build.log http://koji.katello.org/kojifiles/work/tasks/9917/29917/build.log

Reply:
Thanks! I updated